packages/console/src/pages/SignInExperience/tabs/Content/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx

```diff
@@ -50,8 +51,14 @@ function LanguageDetails() {
   const translationEntries = useMemo(
     () =>
       Object.entries((isBuiltIn ? resource[selectedLanguage] : en).translation).filter(
-        // Filter out the demo app phrases in AC
-        ([groupKey]) => groupKey !== 'demo_app'
+        ([groupKey]) => {
```

Comment:
Should implement a more general solution. e.g. A exclusive list.

Reply:
updated.